packages/core/src/di/injector_compatibility.ts

```diff
@@ -126,7 +126,7 @@ Please check that 1) the type for the parameter at index ${
  * @param flags Optional flags that control how injection is executed.
  * The flags correspond to injection strategies that can be specified with
  * parameter decorators `@Host`, `@Self`, `@SkipSef`, and `@Optional`.
- * @returns True if injection is successful, null otherwise.
+ * @returns True if injection is successful, `null` otherwise.
```

Comment:
Well if we are going to fix the `null` then we should fix its alternative return value to...

```suggestion
 * @returns the injected value if injection is successful, `null` otherwise.
```

Reply:
fixed